d3ledger/back-office
# D3 1058| Remove admin functions

# Description
https://soramitsu.atlassian.net/browse/D3-1058

In order to correct work, has changes from https://github.com/d3ledger/back-office/pull/181

# Works done
- [x] Remove fee settings 
- [x] Remove transaction explorer

# How to test 
0. run back-office, notary
1. Login as admin
2. Check that there is no fee settings and explorer page
